ChaosMod/Effects/db/Player/PlayerBusted.cpp

```diff
@@ -0,0 +1,156 @@
+#include <stdafx.h>
+
+/*
+* Effect by kolyaventuri
+*/
+
+enum ArrestState {
+	start = 0,
+	startBust,
+	handsUp,
+	cleanup
+};
+
+static const int stopTime = 300;
+static const char* cops[3] = { "s_m_y_cop_01", "s_f_y_cop_01", "csb_cop" };
+static int currentMode = ArrestState::start;
+static int lastModeTime = 0;
+static int nextModeTime = 0;
+static Object waterObj = 0;
+
+static void OnStart() {
+	Player playerId = PLAYER_ID();
+	Ped player = PLAYER_PED_ID();
+	Vector3 pos = GET_ENTITY_COORDS(player, true);
+	const char* hashKey = cops[g_random.GetRandomInt(0, 2)];
+
+	currentMode = ArrestState::start;
+	lastModeTime = 0;
+	nextModeTime = 0;
+
+	Ped cop = CreatePoolPed(6, GET_HASH_KEY(hashKey), 0.f, 0.f, 0.f, GET_ENTITY_HEADING(player));
```

Comment:
Doesn't have to be in the mod's entity pool since the ped is being removed manually by the effect afterwards, can simply use `CREATE_ENTITY` for that (and make sure to load the model first with `LoadModel`)